src/main/java/com/twentyonec/ItemsLogger/commands/Commands.java

```diff
@@ -0,0 +1,67 @@
+package com.twentyonec.ItemsLogger.commands;
+
+import java.util.regex.Pattern;
+
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import com.twentyonec.ItemsLogger.ItemPlayer;
+import com.twentyonec.ItemsLogger.ItemsLogger;
+import com.twentyonec.ItemsLogger.utils.ChatHandler;
+import com.twentyonec.ItemsLogger.utils.Storage;
+
+import net.md_5.bungee.api.chat.TextComponent;
+
+public class Commands implements CommandExecutor {
+
+	ItemsLogger plugin = ItemsLogger.getPlugin();
+	Storage storage = Storage.getStorage(plugin);
+
+	@Override
+	public boolean onCommand(final CommandSender sender, final Command command, final String label, final String[] args) {
+
+		if (!(sender instanceof Player))
+			return false;
+		if (args.length < 1)
+			return false;
+
+		final Player target = (Bukkit.getServer().getPlayer(args[0]));
+		if (target == null) {
+			sender.sendMessage("§8[§cItemsLogger§8] §e" + args[0] + " is not online!");
+			return false;
+		}
+
+		String date = null;
+		String cause = null;
+		Integer index = null;
```

Comment:
No need to use the object type when the atomic type will do.
```suggestion
		int index;
```